server/broadcast_test.go

```diff
@@ -580,12 +579,8 @@ func TestTranscodeSegment_VerifyPixels(t *testing.T) {
 	// Wait for async pixels verification to finish
 	time.Sleep(1 * time.Second)
 
-	// Check that the session was removed because we are in on-chain mode
-	_, ok = bsm.sessMap[ts.URL]
-	assert.False(ok)
-
 	// Create stub response with correct reported pixels
-	p, err := pixels("test.flv")
+	p, err := verification.Pixels("test.flv")
```

Comment:
I thin this test is basically a series of no-ops right now due to a default policy not being defined, so the verification code never gets executed - see https://github.com/livepeer/go-livepeer/pull/1360#discussion_r376148479 .

I'm thinking that this test can be removed as long as we add something that ensures pixel counting is invoked from the SegmentVerifier.Verify function if a nil verifier is used. There was a test added that does a failure check; we should also check for success as well.

Reply:
Test removed & pixels() function unexported in 3bd8bc22820bcbb5061579b1a30f31bd56447349